resources/charts/fork-observer/templates/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "fork-observer.fullname" . }}
+  labels:
+    {{- include "fork-observer.labels" . | nindent 4 }}
+data:
+  config.toml: |
+    {{- .Values.configQueryInterval | nindent 4 }}
```

Comment:
very nice, I like this pattern: user configurable items that MUST be present get their own key in values.yaml (which allows us to set a default), baseConfig is what we need regardless of user input can be defines as a text string, and config can then be a catch all for everything else , where the user needs to pass in the actual string to concatenate to the config file. all of the logic for creating the final config file that will eventually end up in a configmap can live in the template function, and we can use helms semver features to make sure we always construct a correctly formatted config for whatever version of the thing we are deploying.

this should work regardless of the config file type, e.g. bitcoin.conf , config.toml, etc.

Reply:
Just hooking this up to the `quickstart` command, as why not...

Reply:
Nice! might also be worth adding `max_interesting_heights` as a key values.yaml, as well?

Reply:
Could do ... could do ... 😭

Reply:
Haha either we add it here (and then this looks RFM), or we can merge now and you can open a follow-up issue to fix the hardcoded value?

Reply:
I am not goign to work on this so leave for a followup I guess

Reply:
884d94b3845d7a73d22dc4cc9106af8022900976